mcs/class/System.Net/Test/Tls_Compat/Harness.cs

```diff
@@ -0,0 +1,252 @@
+using System;
+using System.Threading.Tasks;
+using System.Security.Cryptography.X509Certificates;
+using System.IO;
+using System.Net;
+using System.Net.Sockets;
+using System.Net.Security;
+using System.Collections.Generic;
+
+namespace Macaco
```

Comment:
Does this namespace have a special meaning? :smile: 

Btw. this file doesn't follow the [Mono coding conventions](http://www.mono-project.com/community/contributing/coding-guidelines/).